.github/workflows/python-package.yml

```diff
@@ -26,8 +26,8 @@ jobs:
             toxenv: "py38"
           - python-version: 3.9
             toxenv: "py39"
-          - python-version: 2.7
-            toxenv: "py27"
+          - python-version: "3.10"
+            toxenv: "py310"
```

Comment:
Python 3.11 is GA. We are planning to support it in next sprint (https://github.com/Azure/azure-cli/pull/24109).